filebeat/harvester/log_test.go

```diff
@@ -64,7 +64,7 @@ func TestReadLine(t *testing.T) {
 		maxBackoffDuration: 1 * time.Second,
 		backoffFactor:      2,
 	}
-	reader, _ := createLineReader(fileSource{readFile}, codec, 100, 1000, readConfig, nil, nil)
+	reader, _ := createLineReader(fileSource{readFile}, codec, 100, 1000, readConfig, nil, nil, make(chan struct{}))
```

Comment:
one can pass nil if channel is not to be closed. The nil-channel will always block.